man/chpl.txt

```diff
@@ -195,6 +195,12 @@ OPTIONS
   --[no-]bounds-checks   Enable [disable] run-time bounds checking,
                     e.g. during slicing and array indexing.
 
+  --[no-]formal-domain-checks   Enable [disable] run-time checks to ensure
+                    that an actual array argument's domain matches its formal
+                    array argument's domain in terms of (a) describing the same
+                    index set and (b) having an equivalent domain map (if the formal
+                    domain explicitly specifies a domain map).
+
   --[no-]local-checks  Enable [disable] run-time checking of the locality of
                     references within local blocks.
 
```

Comment:
Not related to the man page, but don't want to clutter the global comments just yet: Isn't there a README.dsi that need s to be updated too?